vendor/autoload.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
Usually the `vendor` directory is excluded from Git as it is generated by Composer. Not sure how this will affect packaging of the plugin.

Reply:
TODO:

- Exclude `/vendor/` directory from repo and include it in the first release(some time...).